README.md

```diff
@@ -750,7 +751,7 @@ The following table lists the parameters for the `generate-delta-worker` compone
 | `generate_delta_worker.automigrate` | Enable automatic database migrations at service start up | `true` |
 | `generate_delta_worker.image.registry` | Docker image registry | `registry.mender.io` |
 | `generate_delta_worker.image.repository` | Docker image repository | `mendersoftware/generate-delta-worker` |
-| `generate_delta_worker.image.tag` | Docker image tag | `mender-3.5.0` |
+| `generate_delta_worker.image.tag` | Docker image tag | `nil` |
```

Comment:
there is also this: [tag: mender-3.5.0](https://github.com/mendersoftware/mender-helm/blob/master/mender/values.yaml#L200)

Reply:
> there is also this: [tag: mender-3.5.0](https://github.com/mendersoftware/mender-helm/blob/master/mender/values.yaml#L200)

This is from the `generate_delta_worker` image, not available in `mender-3.4.0` and let disabled in the `values.yaml` by default. So we're overriding the default here.
I know it's not the best solution we have, but it's kind of "feature flag"